packages/terra-date-picker/tests/nightwatch/date-picker-spec.js

```diff
@@ -93,6 +93,39 @@ module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous
     browser.expect.element('h3').text.to.not.contain('2017-07-12');
   },
 
+  'Triggers onChangeRaw when a date input value is changed': (browser) => {
+    browser.url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/date-picker-tests/on-change-raw`);
```

Comment:
With the newest nightwatch changes, brower.url is specified as:

`browser.url('${browser.launchUrl}/#/tests/date-picker-tests/on-change-raw')`

Reply:
Updated.
https://github.com/cerner/terra-core/pull/768/commits/32ba54d38fa0e035681655c69a78110d176ddfca